versions/3.0.4.md

```diff
@@ -89,11 +93,9 @@ The OpenAPI Specification is versioned using a `major`.`minor`.`patch` versionin
 
 Occasionally, non-backwards compatible changes may be made in `minor` versions of the OAS where impact is believed to be low relative to the benefit provided.
 
-An OpenAPI description document compatible with OAS 3.\*.\* contains a required [`openapi`](#oas-version) field which designates the version of the OAS that it uses.
-
 ### Format
 
-An OpenAPI description document that conforms to the OpenAPI Specification is itself a JSON object, which may be represented either in JSON or YAML format.
+An OpenAPI Document that conforms to the OpenAPI Specification is itself a JSON object, which may be represented either in JSON or YAML format.
```

Comment:
Found one more concern while porting to 3.1.1:  3.x OADs are not single JSON objects in any sense that I can map to [RFC8259 §4](https://datatracker.ietf.org/doc/html/rfc8259#section-4).

```suggestion
An OpenAPI Document that conforms to the OpenAPI Specification is itself a set of JSON objects, which may be represented either in JSON or YAML format.
```

Reply:
Wait... never mind, I confused myself.